src/api/demo.rkt

```diff
@@ -350,7 +351,7 @@
 (define (check-status req job-id)
   (match (get-timeline-for job-id)
     [(? hash? result-hash)
-     (response/full 201
+     (response/full 200
```

Comment:
Look, I guess whatever, but in this case I think a 201 does seem more appropriate? Since it doesn't include the body?